components/fieldgroups/shared/current-weather.schema.json

```diff
@@ -124,7 +124,8 @@
                   "$ref": "#/definitions/speed"
                 },
                 "xdm:sunsetTime": {
-                  "type": "integer",
+                  "type": "string",
```

Comment:
Changing a data type of an existing field is breaking change. This is not allowed
cc:  @anandphatak